[ONNX] Add supplementary tests and description for custom_opsets param from torch.onnx.export() function. (#62085)

BowenBao opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #64382 [ONNX] Deprecate use_external_data_format param from torch.onnx.export() function. (#62257)
* #64380 [ONNX] Deprecated the example_outputs param from torch.onnx.export() function. (#62815)
* #64373 [ONNX] minor doc improvements and cleanup (#62514)
* **#64372 [ONNX] Add supplementary tests and description for custom_opsets param from torch.onnx.export() function. (#62085)**
* #64371 [ONNX] Remove strip_doc_string param from torch.onnx.export() function. (#61712)
* #64370 [ONNX] Remove argument _retain_param_name from torch.onnx.export() function. (#61702)

custom_opsets arg from torch.onnx.export() is no needed to be removed.

Add some supplementary description and tests for easier understanding.

Co-authored-by: hwangdeyu <dejack953@outlook.com>

Differential Revision: [D30905269](https://our.internmc.facebook.com/intern/diff/D30905269)

hwangdeyu committed:
[ONNX] Add supplementary tests and description for custom_opsets param from torch.onnx.export() function. (#62085)

custom_opsets arg from torch.onnx.export() is no needed to be removed.

Add some supplementary description and tests for easier understanding.

Co-authored-by: hwangdeyu <dejack953@outlook.com>

[ghstack-poisoned]

BowenBao committed:
Update on "[ONNX] Add supplementary tests and description for custom_opsets param from torch.onnx.export() function. (#62085)"


custom_opsets arg from torch.onnx.export() is no needed to be removed.

Add some supplementary description and tests for easier understanding.

Co-authored-by: hwangdeyu <dejack953@outlook.com>

[ghstack-poisoned]

BowenBao committed:
Update on "[ONNX] Add supplementary tests and description for custom_opsets param from torch.onnx.export() function. (#62085)"


custom_opsets arg from torch.onnx.export() is no needed to be removed.

Add some supplementary description and tests for easier understanding.

Co-authored-by: hwangdeyu <dejack953@outlook.com>

[ghstack-poisoned]

BowenBao committed:
Update on "[ONNX] Add supplementary tests and description for custom_opsets param from torch.onnx.export() function. (#62085)"


custom_opsets arg from torch.onnx.export() is no needed to be removed.

Add some supplementary description and tests for easier understanding.

Co-authored-by: hwangdeyu <dejack953@outlook.com>

[ghstack-poisoned]

msaroufim commented:
@msaroufim has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30905269).

BowenBao committed:
Update on "[ONNX] Add supplementary tests and description for custom_opsets param from torch.onnx.export() function. (#62085)"


custom_opsets arg from torch.onnx.export() is no needed to be removed.

Add some supplementary description and tests for easier understanding.

Co-authored-by: hwangdeyu <dejack953@outlook.com>

Differential Revision: [D30905269](https://our.internmc.facebook.com/intern/diff/D30905269)

[ghstack-poisoned]

malfet commented:
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30905269).

BowenBao committed:
Update on "[ONNX] Add supplementary tests and description for custom_opsets param from torch.onnx.export() function. (#62085)"


custom_opsets arg from torch.onnx.export() is no needed to be removed.

Add some supplementary description and tests for easier understanding.

Co-authored-by: hwangdeyu <dejack953@outlook.com>

Differential Revision: [D30905269](https://our.internmc.facebook.com/intern/diff/D30905269)

[ghstack-poisoned]

malfet commented:
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30905269).
